sdk/webpubsub/Azure.ResourceManager.WebPubSub/CHANGELOG.md

```diff
@@ -1,12 +1,27 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.4 (Unreleased)
```

Comment:
Why do we need this part? `Beta.4` will be automatically added after we release `beta.3`.